package.json

```diff
@@ -20,9 +20,11 @@
     "postcss": "8.4.27",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "typescript": "5.1.6"
+    "typescript": "5.1.6",
+    "zod": "^3.21.4"
   },
   "devDependencies": {
+    "@tanstack/eslint-plugin-query": "^4.32.5",
```

Comment:
this one too

Reply:
also, if you are moving these manually from the file, make sure to run `npm i` so the changes get reflected in `package-lock.json`

Reply:
Uninstalled tanstack query eslint plugin dependencies